microsoftgraph/microsoft-graph-explorer-v2
# Fix: show template variables

## Overview
Request urls with values like {group-id} were not being substituted with the relevant value.

Closes #270 
Closes #220 

### Demo

![image](https://user-images.githubusercontent.com/53429535/68698498-08400980-0592-11ea-88c7-169b2cb0377e.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Select the user by email query
* Notice the url has an email attached
* Sign in to Graph Explorer
* Notice the url has your email attached
